underfs/swift/src/test/java/alluxio/underfs/swift/SwiftOutputStreamTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.swift;
+
+import static org.mockito.Mockito.verify;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+
+/**
+ * Tests for the {@link SwiftOutputStream} class.
+ */
+@RunWith(PowerMockRunner.class)
+public class SwiftOutputStreamTest {
+
+  private static OutputStream sOutputStreamMock;
+  private static HttpURLConnection sHttpConMock;
```

Comment:
We mutate these fields in non-static contexts and they don't need to be static, so lets make them non-static.